mobile/src/actions/settings.js

```diff
@@ -79,6 +81,21 @@ export const registerDevice = () => async (dispatch, getState) => {
     throw err
   }
   initClient(getState().mobile.settings.serverUrl, onRegister(dispatch), getDeviceName())
+
+  let isV2Instance
+  try{
+    isV2Instance = await cozy.client.isV2()
+  }
+  catch (err) {
+    // this can happen if the HTTP request to check the instance version fails; in that case, it is likely to fail again and be caught during the authorize process, which is designed to handle this
+    isV2Instance = false
```

Comment:
We could dispatch a `wrongAddressError` error here, but I think it's preferable not to stop the process because the version detection fails.